BraintreeUIKit/Localization/BTUIKLocalizedString.m

```diff
@@ -2,9 +2,21 @@
 
 @implementation BTUIKLocalizedString
 
-+ (NSBundle *)localizationBundle {
+static NSArray *customTranslations;
 
++ (NSBundle *)localizationBundle {
     static NSString * bundleName = @"Braintree-UIKit-Localization";
+    if ([[NSLocale preferredLanguages] count] > 0) {
+        NSString *language = [[NSLocale preferredLanguages] firstObject];
+        // Ignore region portion of local ID
+        language = [[language componentsSeparatedByString:@"_"] firstObject];
```

Comment:
Slight preference for chaining this into one statement -- I initially interpreted this as a bug where language was getting overwritten. Something like this:

```
language = [[[[language componentsSeparatedByString:@"_"] firstObject] componentsSeparatedByString:@"-"] firstObject];
```